java/src/test/java/com/saucelabs/saucebindings/acceptance/SauceDesktopTest.java

```diff
@@ -47,4 +43,16 @@ public void runsEUCentral() {
         assertNotNull(webDriver);
         assertTrue(session.getSauceUrl().toString().contains("eu-central-1"));
     }
+
+    // Same as US_EAST just more descriptive
+    @Test
+    public void createsHeadlessDriver() {
+        SauceOptions sauceOptions = SauceOptions.headless();
+        session = new SauceSession(sauceOptions);
+        session.setDataCenter(DataCenter.HEADLESS);
```

Comment:
this is a bit weird, can't we set the data center automatically?

Reply:
Yeah, it just added an extra complication. You know, let's just pull this one out for now as a "nice to have" in the future